scam-detector-py/src/agent.py

```diff
@@ -1260,10 +1260,18 @@ def handle_alert(alert_event: forta_agent.alert_event.AlertEvent) -> list:
             persist_state()
             logging.info(f"{BOT_VERSION}: Persisted state")
 
-        for finding in FINDINGS_CACHE_ALERT[0:10]:  # 10 findings per handle alert due to size limitation
-            if finding is not None:
-                findings.append(finding)
-        FINDINGS_CACHE_ALERT = FINDINGS_CACHE_ALERT[10:]
+        if CHAIN_ID == 1:
```

Comment:
not quite shure what this change is about. Can you add a comment why this is necessary?

Reply:
makes sense now; i would add a comment around why only on chain 1 and rename to filter out likely fps. 
I would also add a comment around why this is done in this function as opposed to when the finding was first created.

Reply:
do we want to remove the finding completely? I suggest removal for prod, but still emit it in beta, so we can get a sense on how much is filtered out.

Reply:
@christian-forta, Attack and Scam detector have an `alert_FP`, but it's used to remove labels after an FP has been triggered. In this case Etherscan prevents the FP, so I thought of creating a separate alert. 
Please let me know if I misunderstood anything, and I'll update the logic.

Regarding the tests, consuming the manual lists, taking too much, because `TEST_STATE` was not properly set, I put the "setting test state" logic in the `initialize`.